sequence_processing_pipeline/QCJob.py

```diff
@@ -202,6 +202,11 @@ def _process_sample_sheet(self):
 
         header = valid_sheet.Header
         chemistry = header['chemistry']
+
+        if header['Assay'] not in ['TruSeq HT', 'Metagenomic']:
```

Comment:
Given the list of valid assays is used here, and in Pipeline.py:218, can it be centralized as a variable and imported?

Reply:
great idea thanks!